docs/migration/README.md

```diff
@@ -0,0 +1,152 @@
+* Secret TUF_ON_CI_TOKEN (this is a sigstore-bot token)
+
+#### Update GCP configuration
+
+The settings should be easy to copy from root-signing-staging, with the exception that there are now two different
+service accounts: one for KMS and one for GCS
+* KMS: Online signing must be possible from the online-sign workflow from the main branch
+* GCS: uploading must be possible from publish workflow from the publish branch
+
+#### Update targets/ directory in git
+
+Currently targets/ does not quite match the repository contents.
+
+#### Add tuf-on-ci workflows to git
+
+Add the workflows but disable scheduled runs and specifically prevent publishing before it is enabled.
+
+#### Add import configuration to git
+
+* Add import script that copies metadata in expected locations and rewrites whitespace (so future diffs are manageable)
+* Add configuration file for the actual tuf-on-ci import
+
+#### Add/Update playbooks to git
+
+This playbook and manuals for signers and maintainers.
+
+#### Signer orientation
+
+Make sure signers have installed and tested the software, and understand what will be requested during the signing event
+
+#### General outreach
+
+Communicate the upcoming changes to Sigstore community
+
+### 2. Initial signing event
+
+The initial signing event can be restarted if it looks like there are issues that cannot be solved within the ~3 day timeline.
+It makes sense to start the signing event just after online signing to make that window as large as possible.
+
+#### Start signing event
+
+* Maintainer: Run the import script to copy current metadata to new directory, merge to main
+* Maintainer: run tuf-on-ci-import-repo with the prepared configuration file: this starts a signing event
+* Maintainer: run tuf-on-ci-delegate to modify online roles in the signing event:
+  * they should use the same key (this makes sense in root-signing as both snapshot and timestamp are
+    signed in same process with same access credentials)
+  * signing period is 4 days, expiry 7 days
+
+#### Signing event
+
+Every signer in the repository needs to sign the signing event at this point by running `tuf-on-ci-sign`:
+this is required because the keyids of all keys have changed (the actual key content has not).
+
+** TODO: This includes the npmjs signer: it is unclear how this signing happens at this point**
+
+In the same signing event PR all workflows can be enabled. Note that:
+* deploy-to-gcs step should be still left commented in publish workflow
+* the legacy workflows can keep running (although online-sign can be disabled if schedule is tight )
+
+#### First online sign
+
+Merging the signing event PR
+* triggers online signing
+* publishes the test "preprod" repository sigstore.github.io/root-signing/
+* runs all client tests against that repository
+* does NOT publish to production GCS yet as this is still disabled
+
+Manual client tests against this repository are now possible.
```

Comment:
```suggestion
Manual client tests against this repository are now possible and should be completed with several clients before proceeding.
```

?

if so, which clients?

Reply:
we have automated tests with cosign, sigstore-go , -python, -js and -java. 

I don't know what manual testing we should really _require_ here.